### PR TITLE
Fix lint for web source

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

Run `yarn lint:fix` on web source

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/web/src/components/applications-page/add-application-drawer/index.test.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ afterAll(() => {
 });
 
 describe("AddApplicationDrawer", () => {
-  it("should pass", () => { });
+  it.todo("should pass");
```
Comment (nghialv) [inline pkg/app/web/src/components/applications-page/add-application-drawer/index.test.tsx:18]: Nice. Didn't know that.

**Comment by nghialv** [comment]
It would be nice if we can have `make lint-web` command.

File: pkg/app/web/src/components/applications-page/add-application-drawer/index.test.tsx (hunk 1 above)
Comment (khanhtc1202) [reply pkg/app/web/src/components/applications-page/add-application-drawer/index.test.tsx:18]: Me too 😂

**Comment by khanhtc1202** [comment]
> It would be nice if we can have `make web-lint` command.

Lets add it by separated PR

**Pull request merged**